### ISSUE TITLE
Question: Is there a reason a job is saved when it is enqueued?

### ISSUE THREAD

**Issue by phil-monroe** [ondrejbartas/sidekiq-cron#141]
Looking at the code, when a job is `enque!`'d, it is also re-saved back into redis. Is there a reason for this? It seems like a few round trips to redis that are un-necessary and an `info` level log message for something that happens often. In our case, it happen every minute for a frequently recurring job, which is bloating our logs. 

I'm happy to submit a PR for any possible optimizations, but just want to understand the reasoning before diving in. Thanks!

https://github.com/ondrejbartas/sidekiq-cron/blob/master/lib/sidekiq/cron/job.rb#L71
``` ruby
      #enque cron job to queue
      def enque! time = Time.now
        @last_enqueue_time = time

        klass_const =
            begin
              @klass.to_s.constantize
            rescue NameError
              nil
            end

        if klass_const
          if defined?(ActiveJob::Base) && klass_const < ActiveJob::Base
            enqueue_active_job(klass_const)
          else
            enqueue_sidekiq_worker(klass_const)
          end
        else
          if @active_job
            Sidekiq::Client.push(active_job_message)
          else
            Sidekiq::Client.push(sidekiq_worker_message)
          end
        end

        save
        logger.debug { "enqueued #{@name}: #{@message}" }
      end

# ...
      def save
        #if job is invalid return false
        return false unless valid?

        Sidekiq.redis do |conn|

          #add to set of all jobs
          conn.sadd self.class.jobs_key, redis_key

          #add informations for this job!
          conn.hmset redis_key, *hash_to_redis(to_hash)

          #add information about last time! - don't enque right after scheduler poller starts!
          time = Time.now
          conn.zadd(job_enqueued_key, time.to_f.to_s, formated_last_time(time).to_s) unless conn.exists(job_enqueued_key)
        end
        logger.info { "Cron Jobs - add job with name: #{@name}" }
      end
```

**Comment by ondrejbartas** [comment]
you should disable debug logging. And I will update saving only of change value of last_enqueue_time

**Comment by ondrejbartas** [comment]
#153

**Comment by phil-monroe** [comment]
@ondrejbartas: I have disabled debug logging. It is the following line in `save` that pollutes the INFO log:
```
logger.info { "Cron Jobs - add job with name: #{@name}" }
```

That said, your fix in #153 seems perfect. Thanks!

**Comment by ondrejbartas** [comment]
btw difference of performance is 6 seconds on enqueue of 10 000 jobs :D (from 26s to 19s) thanks for fix

**Comment by ondrejbartas** [comment]
released 0.6.0
